K0:
    [458.654,   0.0,  367.215,
    0.0,   457.296,   248.375,
     0.0,   0.0,   1.0]
R0: [0.999966347530033, -0.001422739138722922, 0.008079580483432283, 0.001365741834644127, 0.9999741760894847, 0.007055629199258132, -0.008089410156878961, -0.007044357138835809, 0.9999424675829176]
D0: [-0.28340811, 0.07395907, 0.00019359, 1.76187114e-05, 0.0]
P0: [435.2046959714599, 0, 367.4517211914062, 0, 0, 435.2046959714599, 252.2008514404297, 0, 0, 0, 1, 0]
K1: [457.587, 0.0, 379.999, 0.0, 456.134, 255.238, 0.0, 0.0, 1]
R1: [0.9999633526194376, -0.003625811871560086, 0.007755443660172947, 0.003680398547259526, 0.9999684752771629, -0.007035845251224894, -0.007729688520722713, 0.007064130529506649, 0.999945173484644]
D1: [-0.28368365, 0.07451284, -0.00010473, -3.555907e-05, 0.0]
P1: [435.2046959714599, 0, 367.4517211914062, -47.90639384423901, 0, 435.2046959714599, 252.2008514404297, 0, 0, 0, 1, 0]
p0: [435.2046959714599, 0, 367.4517211914062, 0, 435.2046959714599, 252.2008514404297, 0, 0, 1] 
p1: [435.2046959714599, 0, 367.4517211914062, 0, 435.2046959714599, 252.2008514404297, 0, 0, 1] 
cam0_topic: /cam0/image_raw
cam1_topic: /cam0/image_raw
vo_odom_topic: /odom
path_topic: /path
